lib/ip.go

```diff
@@ -36,13 +37,56 @@ func RandIP4List(n int) []net.IP {
 	return ips
 }
 
+// RandIP4Range returns a list of randomly generated IPv4 addresses within
+// starting and ending IPrange.
+
+// note: starting and ending IPs should be in format of "1.1.1.1"
+func RandIP4Range(startIP, EndIP string) (net.IP, error) {
+	StartIPRaw := net.ParseIP(startIP).To4()
```

Comment:
For all variables local to a function in golang, keep their first letter lowercase, otherwise it looks as if some public/global variable is being used. So `StartIPRaw` should be `startIPRaw`, and the same goes for other variables here.